src/forms/FormStateApp.tsx

```diff
@@ -136,7 +137,14 @@ export function FormStateApp() {
             </div>
           </header>
           <div>
-            <Button label="Read Only" onClick={() => setReadOnly(!readOnly)} />
+            <div>
+              <Button label="Read Only" onClick={() => setReadOnly(!readOnly)} />
+            </div>
+
+            <div css={Css.mt1.$}>
+              <strong>Form Values:</strong>
+              <pre>{JSON.stringify(formState.value, null, 2)}</pre>
+            </div>
```

Comment:
Printing out the Form State's values as it changes. Found this to be helpful when playing around and understanding what data is stored in formState.

Reply:
Nice addition!